CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Do not propage group config (timestamp, expresisonLanguage) to autoprovision device (#1391)
```

Comment:
```suggestion
- Fix: do not propagate group config (timestamp and explicitAttrs) to autoprovisioned devices (at database level) (iotagent-node-lib#1391)
```

Note that expressionLanguage was removed in a recent version, so it doens't belong to the existing API or DB model.